### PR TITLE
Homework lesson 4

### PR DESCRIPTION
**Pull request by CjVizer**
I hope everything is fine with the code :)

### CODE REVIEW

File: Lesson_4/hw_2.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#     ["O", "O", "O"],
+#     ["O", "X", "X"],
+#     ["E", "X", "X"]
+# ]))  # ➞ "O"
+#
+# print(tic_tac_toe([
+#     ["X", "X", "O"],
+#     ["O", "O", "X"],
+#     ["X", "X", "O"]
+# ]))  # ➞ "Draw"
```
Comment (katevoskresenska) [inline Lesson_4/hw_2.py:131]: Everything works as a charm!

File: Lesson_4/hw_2.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        print(f'{color_red}Incorrect input!\033[0m\n')
+    else:
+        player_choice = player_choice.title()
+        computer_choise = choice(g_attr)
+        print('Hmmm...Let me think', end='')
+        for _ in range(5):
+            sleep(0.5)
+            print('.', end='')
+        print(f' {color_green}{computer_choise}{color_end}')
+        if g_attr[g_attr.index(player_choice)] == g_attr[g_attr.index(computer_choise)]:
```
Comment (mmykhailenko) [inline Lesson_4/hw_2.py:24]: if you should repeat your code, better to save it in variable and then use it indx = g_attr.index(player_choice)

File: Lesson_4/hw_2.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+# Do you have enough toilet paper(TP) to make it through?
+# Although the number of squares per roll of TP varies significantly, we'll assume each roll has 500 sheets,
+# and the average person uses 57 sheets per day.
+
+# Create a function that will receive a dictionary with two key/values:
+# "people" ⁠— Number of people in the household.
+# "tp" ⁠— Number of rolls.
+# Return a statement telling the user if they need to buy more TP!
+def more_tp(data):
+    people, tp = data.get('people', None), data.get('tp', None)
```
Comment (mmykhailenko) [inline Lesson_4/hw_2.py:55]: get('people', None) -> get('people') No need to write None

**Review by mmykhailenko — APPROVED** [review]

**Review by mmykhailenko — APPROVED** [review]
Nice

**Pull request merged**